src/rendering/renderers/shared/background/BackgroundSystem.ts

```diff
@@ -97,11 +97,11 @@ export class BackgroundSystem implements System
     {
         options = { ...BackgroundSystem.defaultOptions, ...options };
 
+        const optionsHasBackgroundColor = !!(options.background || options.backgroundColor);
+
         this.clearBeforeRender = options.clearBeforeRender;
         this.color = options.background || options.backgroundColor || this._backgroundColor; // run bg color setter
-        this.alpha = options.backgroundAlpha;
-
-        this._backgroundColor.setAlpha(options.backgroundAlpha);
```

Comment:
Line `104` here was extraneous, removed as alpha logic is already handled by line above.